samples/sqltransport/store-and-forward/Version_5/Sender/Program.cs

```diff
@@ -35,11 +34,12 @@ static void Main()
                     return;
                 }
                 string orderId = new string(Enumerable.Range(0, 4).Select(x => letters[random.Next(letters.Length)]).ToArray());
-                bus.Publish(new OrderSubmitted
+                bus.Defer(TimeSpan.FromSeconds(1),  new OrderSubmitted
```

Comment:
This complains that events should be Published (NSB best practices check?). So failing at runtime.